Detectors/TPC/qc/src/Helpers.cxx

```diff
@@ -0,0 +1,50 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <assert.h>
+
+#include "TPCQC/Helpers.h"
+
+using namespace o2::tpc::qc;
+
+//______________________________________________________________________________
+std::vector<double> helpers::makeLogBinning(const int nbins, const double min, const double max)
+{
+  assert(min > 0);
+  assert(min < max);
+
+  std::vector<double> binLim(nbins + 1);
+
+  const double expMax = std::log(max / min);
+  const double binWidth = expMax / nbins;
+
+  binLim[0] = min;
+  binLim[nbins] = max;
+
+  for (Int_t i = 1; i < nbins; ++i) {
+    binLim[i] = min * std::exp(i * binWidth);
+  }
+
+  return binLim;
+}
+
+//______________________________________________________________________________
+void helpers::setStyleHistogram1D(TH1& histo)
+{
+  histo.SetStats(1);
+}
```

Comment:
It might be nice to have a function which takes a vector of histograms. This will be used in many places I guess. Same for 2D.

Reply:
Only managed to implement a function that takes std::vector<TH2F>* instead of std::vector<TH2>*. I will figure out how o generalize this or implement a function for every single TH2 variant.

Reply:
Same of course for TH1.

Reply:
just noticed that formatting went wrong in my older comment. I implemented the function for a std::vector* of TH2F and TH1F but not generalized for all TH1 and TH2 variants.